common/src/main/java/net/opentsdb/data/LowLevelMetricData.java

```diff
@@ -0,0 +1,68 @@
+// This file is part of OpenTSDB.
+// Copyright (C) 2020  The OpenTSDB Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package net.opentsdb.data;
+
+/**
+ * An extension of the low level time series data that contains metric information.
+ * 
+ * @since 3.0
+ */
+public interface LowLevelMetricData extends LowLevelTimeSeriesData{
+  
+  /** The format of the metrics data. */
+  public static enum ValueFormat {
```

Comment:
Seems to be missing UINTEGER: https://docs.influxdata.com/influxdb/v2.0/reference/syntax/line-protocol/#uinteger

Reply:
Also missing BOOLEAN: https://docs.influxdata.com/influxdb/v2.0/reference/syntax/line-protocol/#boolean

Reply:
Resolution to above: For now, we are intentionally declining to represent this data exactly, and, instead, we intend to transform them into Yamas equivalents (uint => double, bool => 0|1).